packages/core/database/src/entity-manager/relations/cloning/regular-relations.ts

```diff
@@ -73,13 +77,13 @@ const cloneRegularRelations = async ({
     .toSQL();
 
   // Insert the clone relations
-  await strapi.db.entityManager
+  await db.entityManager
     .createQueryBuilder(joinTable.name)
     .insert(
-      strapi.db.connection.raw(
+      db.connection.raw(
         `(${columns.join(',')})  ${selectStatement.sql}`,
         selectStatement.bindings
-      )
+      ) as any
```

Comment:
Ideally, the argument for the insert function should be of type Record, but using Record does not allow for the implementation of insert select. Hence, as a temporary measure following the original implementation, a string type is forcibly used here (though it works).